src/cosalib/gcp.py

```diff
@@ -98,4 +103,12 @@ def gcp_cli(parser):
     parser.add_argument("--project",
                         help="GCP Project name",
                         default=os.environ.get("GCP_PROJECT_NAME"))
+    parser.add_argument("--fcos",
+                        help="""Flag this is Fedora CoreOS (or a derivative);
+                                Currently enables SECURE_BOOT and UEFI_COMPATIBLE""",
+                        action="store_true",
+                        default=False)
```

Comment:
Why have this be off by default?

Reply:
See also https://github.com/openshift/installer/pull/2921 which merged.

Reply:
assuming we want to have SECURE_BOOT and UEFI_COMPATIBLE everywhere we could just get rid of the flag altogether and bake it in at a lower level.

Reply:
though I'd prefer to do this in a follow up if you don't mind.

Reply:
> assuming we want to have SECURE_BOOT and UEFI_COMPATIBLE everywhere we could just get rid of the flag altogether and bake it in at a lower level.

That's exactly the status quo today, it's baked into `ore` by default but you went out of the way to explicitly disable it.

Reply:
Nevermind it's off by default, I was wrong.

Reply:
https://github.com/coreos/coreos-assembler/blob/master/mantle/cmd/ore/gcloud/upload.go#L54

Reply:
follow-up to make `SECURE_BOOT` and `UEFI_COMPATIBLE` standard and remove the `--fcos` option: https://github.com/coreos/coreos-assembler/pull/1333